testdata/integration/simple/upstream/upstream.go

```
@@ -28,3 +30,25 @@ func dereference() {
 	var v *int
 	print(*v) //want "unassigned variable `v` dereferenced"
 }
+
+func DerefNoLintLine(v *int) {
```

golangci-lint (1.64.8)

[warning] exported: exported function DerefNoLintLine should have comment or be unexported

(revive)